tests/std/tests/GH_003131_tuple_opeq_overload_and/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.

```

Comment:
This new test won't run in the internal test machinery if it's not added to `tests/std/test.lst`.